### DISCUSSION TITLE
Long alignment time

### DISCUSSION THREAD

**Discussion by leifnel** [jomjol/AI-on-the-edge-device#2644] [General]
How come alignment takes over two minutes, while decoding only takes a few seconds?

The camera doesn't move that much around.

I have tilted the image a few degrees to compensate for the slant of the _italic digits_, it seems to work better this way.

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/9819287/ee455051-76a2-46ab-818f-97e4b66882ff)

**Comment by Slider0007** [comment]
Alignment time goes quadratic with alignment marker size. Your marker are quite big therefore it takes some time. If you reduce the sizes, evaluation will be quicker.
